abstractions/typescript/tsconfig.es.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "./tsconfig.base.json",
+	"compilerOptions": {
+		"module": "es6",
+		"target": "es6",
```

Comment:
could we at least bump that to es2017? https://github.com/microsoft/kiota/issues/272#issuecomment-885614965

Reply:
setting target to 2017.